ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -506,3 +506,33 @@ def validate_unprivileged_access(self):
             )
         else:
             raise UnexpectedODFAccessException
+
+    def verify_odf_without_ocs_in_installed_operator(self) -> bool:
+        """
+        Function to validate either ODF operator is present post ODF installation,
+        expectation is only ODF operator should be present in Installed operators tab and
+        OCS operator shouldn't be present. This function is only written for 4.9+ versions
+
+        Returns:
+        True: If only odf operator is present in the UI
+        False: If ocs operator is also present in the UI
+        """
+        logger.info("Navigating to Installed Operator Page")
+        self.navigate_installed_operators_page()
+        logger.info("Searching for Openshift Data Foundation Operator")
+        odf_opeartor_presence = self.wait_until_expected_text_is_found(
+            locator=self.validation_loc["odf-operator"],
+            timeout=1,
+            expected_text="OpenShift Data Foundation",
+        )
+        logger.info("Searching for Openshift Container Storage Operator")
+        ocs_opeartor_presence = self.wait_until_expected_text_is_found(
+            locator=self.validation_loc["ocs-operator"],
+            timeout=1,
+            expected_text="OpenShift Container Storage",
+        )
+        if odf_opeartor_presence:
+            if not ocs_opeartor_presence:
+                return True
```

Comment:
can combine to 1 condition?
```
return (odf_opeartor_presence and not ocs_opeartor_presence)

```

Reply:
Good suggestion, @shivamdurgbuns Pls check